tests/checkwhitespace.sh

```diff
@@ -7,7 +7,7 @@ grep -v SAI/ |
 grep -v debian/ |
 grep -v _wrap.cpp |
 perl -ne 'print if /\.(c|cpp|h|hpp|am|sh|pl|pm|install|dirs|links|json|ini|yml|pws|md|py|cfg|conf|i|ac)$/' |
-xargs grep -P "\\s\$"
+xargs grep -nrP "\\s\$"
```

Comment:
why -r is added here ?why you need recursive if find in line 5 is only searching for files not directories

Reply:
Fix it.